### PR TITLE
[ALLUXIO-2136] Mark class MetricsConfigTest as final

### PR DESCRIPTION
**Pull request by Chinese-xingyi**
https://alluxio.atlassian.net/browse/ALLUXIO-2136
To prevent accidental extension.

### CODE REVIEW

**Comment by aaudiber** [comment]
ok to test

**Comment by aaudiber** [comment]
Jenkins, add to whitelist

**Comment by aaudiber** [comment]
@Chinese-xingyi It looks like this PR doesn't actually make any changes~

File: core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
 /**
  * Unit tests for {@link MetricsConfig}.
  */
-public class MetricsConfigTest {
+final public class MetricsConfigTest {
```
Comment (gpang) [inline core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java]: This should be `public final`

**Comment by gpang** [comment]
Jenkins, test this please

**Pull request merged**